manifests/w/WiresharkFoundation/Wireshark/4.0.0/WiresharkFoundation.Wireshark.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 4.0.0
```

```suggestion
PackageVersion: 4.0.0.0
```
This must match Control Panel.